client/my-sites/domains/domain-management/name-servers/style.scss

```diff
@@ -5,12 +5,24 @@
 }
 
 .name-servers__dns {
-	.name-servers__toggle {
-		display: inline-block;
-		float: right;
+	display: flex;
+	flex-direction: column;
 
-		.components-toggle-control .components-base-control__field .components-form-toggle {
-			margin-right: 0;
```

Comment:
Is there a reason why we're removing the right margin? This seems to push the toggle a bit to the left.